content/en/status.md

```diff
@@ -5,21 +5,17 @@ menu:
 aliases: [/project-status, /releases]
 ---
 
-<section class="row td-box td-box--1 position-relative td-box--gradient td-box--height-auto">
-  <div class="container text-center td-arrow-down">
-    <h1>OpenTelemetry Tracing Specification now 1.0!</h1>
-    <p class="h4 mb-0">
-      Our goal is to provide a generally available, production quality
-      release for the tracing data source across most OpenTelemetry components
-      in the first half of 2021. Several components have already reached this
-      milestone! We expect metrics to reach the same status in the second half
-      of 2021 and are targeting logs in 2022.
-    </p>
-  </div>
-</section>
+{{% blocks/lead color="primary" %}}
+# {{% param title %}}: OpenTelemetry Tracing Specification now 1.0!
```

Comment:
Note the title change @austinlparker. I decided to keep the existing inlined title as if it were a subtitle. In anycase @tedsuo has a new version of this page in the works so I guess that this change is temporary :).